src/soundjs/Sound.js

```diff
@@ -1191,23 +1192,22 @@ this.createjs = this.createjs || {};
 	s._parsePath = function (value) {
 		if (typeof(value) != "string") {value = value.toString();}
 
-		/*var match = value.match(s.FILE_PATTERN);
-		if (match == null) {return false;}
-
-		var name = match[4];
-		var ext = match[5];*/
 		//Changed how extensions and filenames are grabbed from the url to add simple support for
 		//urls that use php services & query string params to specify the file
-		var ext, name, origExt;
-		if(value.indexOf(".") >= 0)
-		{
-			ext = value.substr(value.lastIndexOf(".")+1);
-			if(ext.indexOf("?") > 0)
-				ext = ext.substring(0, ext.indexOf("?"));
-			name = value.substring(0, value.lastIndexOf("."));
-			if(name.indexOf("/") >= 0)
-				name = name.substr(name.lastIndexOf("/")+1);
-			origExt = ext;
+		var match = value.match(s.FILE_PATTERN);
+		if (match === null) {return false;}
+
+		var name = '';
+		var ext = '';
+		var origExt = '';
+		if (s.SUPPORTED_EXTENSIONS.indexOf(match[5]) >= 0) {
```

Comment:
It seems that the code in this file keeps the brackets in the same line, the original line 1203 being an exception. I'm happy to change it, but I just want to make certain since dropping it down will be inconsistent with the rest of this file.